Remotion/Data/DomainObjects.UnitTests/Persistence/Rdbms/SqlServer/IntegrationTests/SqlProviderGeneratedSqlTestHelper.cs

```diff
@@ -42,9 +42,10 @@ public SqlProviderGeneratedSqlTestHelper (IStorageSettings storageSettings, Rdbm
       _provider = RdbmsProviderObjectMother.CreateForIntegrationTest(
           storageSettings,
           rdbmsProviderDefinition,
-          (providerDefinition, persistenceListener, commandFactory) =>
+          (providerDefinition, connectionString, persistenceListener, commandFactory) =>
               new ObservableRdbmsProvider(
                   providerDefinition,
+                  connectionString,
```

Comment:
here, we can use `providerDefinition.ConnectionString` since we have the whole context available to decide this.